src/EventListener/SendEventOnRetriedMessage.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SymfonyCasts\MessengerMonitorBundle\EventListener;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\Messenger\Event\WorkerMessageFailedEvent;
+use Symfony\Component\Messenger\EventListener\SendFailedMessageForRetryListener;
+use Symfony\Component\Messenger\Stamp\SentToFailureTransportStamp;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use SymfonyCasts\MessengerMonitorBundle\FailedMessage\MessageRetriedByUserEvent;
+use SymfonyCasts\MessengerMonitorBundle\Stamp\MonitorIdStamp;
+
+/**
+ * @internal
+ */
+final class SendEventOnRetriedMessage implements EventSubscriberInterface
+{
+    private $decorated;
+    private $eventDispatcher;
+    private $logger;
+
+    public function __construct(SendFailedMessageForRetryListener $decorated, EventDispatcherInterface $eventDispatcher, LoggerInterface $logger = null)
```

Comment:
This is a clever solution. But it still feels like we should *also* consider sending a PR to Symfony for an extra hook point (but keep this decoration in our bundle, so that things are working)

Reply:
ok, i'm gonna do that as well